events/utils.py

```diff
@@ -38,3 +44,38 @@ def update(d, u):
         else:
             d[k] = u[k]
     return d
+
+
+def recache_n_events(keywords):
+    """
+    Recache the number of events for the given keywords.
+
+    :type keywords: Iterable[Keyword]
```

Comment:
Since there's `keywords.update()` there now, `Iterable[Keyword]` is a lie; it has to be a `QuerySet[Keyword]`.